client/blocks/edit-gravatar/index.jsx

```diff
@@ -44,7 +45,8 @@ export class EditGravatar extends Component {
 		super( ...arguments );
 		this.state = {
 			isEditingImage: false,
-			image: false
+			image: false,
+			showEmailVerificationNotice: false,
 		};
 	}
```

Comment:
If we don't need any props for default state, we can avoid the constructor here.

```
state = {
    isEditingImage: false,
    image: false,
    showEmailVerificationNotice: false,
};
```